### PR TITLE
Add hiera support for resource

### PR DESCRIPTION
**Pull request by xcompass**
Allow setup resources within hiera

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -19,7 +19,9 @@
   $option_static_route = undef,
   $options            = undef,
   $authoritative      = false,
-  $dhcp_root_group    = $dhcp::params::root_group
+  $dhcp_root_group    = $dhcp::params::root_group,
+  $pools              = {},
+  $hosts              = {}
```
Comment (ekohl) [inline manifests/init.pp]: Nowadays we add trailing commas so I know it wasn't there before, but mind adding it while you're here?

**Comment by ekohl** [comment]
Looks like a simple enough change so other than the trivial nitpick :+1:

**Comment by xcompass** [comment]
@ekohl updated

**Comment by ekohl** [comment]
:+1:

**Comment by mmoll** [comment]
merged, thanks @xcompass!

**Pull request merged**